tests/tools/tflite_run/src/tflite_run.cc

```diff
@@ -37,6 +37,9 @@
 #include <algorithm>
 #include <vector>
 
+#ifdef __clang__
+#include <libgen.h>
+#endif
```

Comment:
As we copy and pass `char *` below instead of `const char *`,

```suggestion
#include <libgen.h>
```